documentation/docs/04-hooks.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Hooks
+---
+
+An optional `src/hooks.js` (or `src/hooks.ts`, or `src/hooks/index.js`) file exports three functions, all optional, that run on the server — **getContext**, **getSession** and **handle**.
+
+> The location of this file can be [configured](#configuration) as `config.kit.files.hooks`
+
+### getContext
+
+This function runs on every incoming request, and generates the `context` property passed to [endpoint handlers](#routing-endpoints) and used to derive the [`session`](#hooks-getsession) object available in the browser.
```

Comment:
It's getting passed as one of the keys in the request object, right? I don't think wording things this way is _wrong_, but I also don't think it's as clear as it could be.

Reply:
agree, have clarified it a bit

Reply:
Yep, sounds better now.